pulp_container/app/viewsets.py

```diff
@@ -1046,7 +1046,7 @@ class ContainerDistributionViewSet(DistributionViewSet):
                 "action": ["pull"],
                 "principal": "*",
                 "effect": "allow",
-                "condition": [
+                "condition_expression": [
```

Comment:
The update of that package will happen in pulpcore, right? Would that need a migration?

Reply:
yes, the new version of drf-access-policy was already merged in pulpcore. I don't think however it'd need a migration, since the change happened on the viewset, the policy will be properly updated here https://github.com/pulp/pulpcore/blob/master/pulpcore/app/apps.py#L197

Reply:
to five some background, tldr they have split the condition element into condition and condition_expression, where the latter one should be use during the boolean logic evaluation

Reply:
I was thinking about the non default policies (modified by users). But when you say, that it's not as easy as moving "condition" to "condition_expression" we should not even think about touching it in a migration.

Reply:
hm, we're not supposed to touch customized policies, that's why we're exposing this change in the changelog entry so users can take action if needed

Reply:
actually @newswangerd  fyi

Reply:
wow, this is annoying. Thanks for the heads up @ipanova